src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -25,7 +25,7 @@ public function __construct(CachedLabelsApi $labelsApi)
     public function onPullRequest(GitHubEvent $event)
     {
         $data = $event->getData();
-        if ('opened' !== $action = $data['action']) {
+        if (!in_array($action = $data['action'], array('opened', 'edited'), true)) {
```

Comment:
We could use a class to hold actions as constants like events. What do you think?